manifests/init.pp

```diff
@@ -13,6 +13,8 @@
   Boolean $manage_package = $::riak::params::manage_package,
   Boolean $manage_repo    = $::riak::params::manage_repo,
   String[1] $version      = $::riak::params::version,
+  $ulimits_nofile_soft    = $::riak::params::ulimits_nofile_soft,
+  $ulimits_nofile_hard    = $::riak::params::ulimits_nofile_hard,
```

Comment:
We should type these parameters. Are we expecting this to be an integer or string? I see ulimits set as a string in one place and an int in the other. If string, we'd use something like `String[1] $ulimits_nofile_soft` to indicate a string of length at least one. For integer, it would be `Integer $ulimits_nofile_soft`.